internal/core/operations/add_rooms/add_rooms_definition.go

```diff
@@ -38,7 +38,11 @@ type AddRoomsDefinition struct {
 }
 
 func (d *AddRoomsDefinition) ShouldExecute(_ context.Context, _ []*operation.Operation) bool {
-	return true
+	if d.Amount <= 10 {
```

Comment:
nit: we can do just the return here `return d.Amount <= 10` instead of the if/else clause.

Reply:
🤦

Reply:
014ab84